internal/worker/server.go

```diff
@@ -289,6 +361,15 @@ func (s *Server) FinishJob(token uuid.UUID, result json.RawMessage) error {
 		}
 	}
 
+	// var jobResult OSBuildJobResult
+	_, _, err = s.JobStatus(jobId, &result)
+	if err != nil {
+		return fmt.Errorf("error finding job status: %v", err)
+	}
+
+	_ = getStatusCode(result)
```

Comment:
Hm not sure if we should store the status code in the jobqueue. The error will always be in the jobresult and is always able to be interpreted by composer

Reply:
Oh the idea wasn't to store it in the job queue, that's just where we are collecting the metrics:
https://github.com/osbuild/osbuild-composer/blob/main/internal/jobqueue/dbjobqueue/dbjobqueue.go#L366-L367

Reply:
ah I wonder if that could move to the worker server when finishing the job up.

Reply:
Yeah we can try that. I'll move this to a separate PR and do that after this one lands